docs/plugins/zentao.md

```diff
@@ -75,3 +75,37 @@ tools:
         # nodePort of zentao service, currently zentao plugin only support `nodePort` type
         nodePort: 30081
 ```
+
+## Deployment
+
+### Step1: Prepare a Kubernetes Cluster
+- If you already have a kubernetes cluster, ignore this step. 
+- If not, you can use `hack/e2e/e2e-up.sh` to create a k8s cluster via `Kind` as test environment.
```

Comment:
k8s -> K8s, let's follow the official convention for abbreviations.

Reply:
Done